src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i]['type']) {
+      case 'addProperties':
+        Object.assign(state, actions[i]['extraData']);
+        break;
```

Comment:
I think it would be clearer and more beautiful:
```suggestion
  for (const action of actions) {
    switch (action.type) {
      case 'addProperties':
        Object.assign(state, action.extraData);
        break;
```

Reply:
Согласен, красивее и понятней - исправляю!